sdk/core/azure-core-test/src/main/java/com/azure/core/test/TestBase.java

```diff
@@ -53,6 +53,11 @@ public abstract class TestBase implements BeforeEachCallback {
      */
     public static final String AZURE_TEST_SERVICE_VERSIONS_VALUE_ALL = "ALL";
 
+    /**
+     * Specifies that the out of process test proxy should be used.
+     */
+    protected static boolean enableTestProxy;
```

Comment:
Should this be package-private instead of protected to ensure it can't be modified outside of `TestProxyBaseTest`